examples/version-migration/separate-container/src/start.ts

```diff
@@ -77,84 +132,51 @@ const renderModel = (model: IVersionedModel, migrationTool: IMigrationTool): voi
 	}
 };
 
-async function start(): Promise<void> {
-	const loader = new SimpleLoader({
-		urlResolver: new InsecureTinyliciousUrlResolver(),
-		documentServiceFactory: new RouterliciousDocumentServiceFactory(
-			new InsecureTinyliciousTokenProvider(),
-		),
-		codeLoader: new DemoCodeLoader(),
-		generateCreateNewRequest: createTinyliciousCreateNewRequest,
+export const setupContainer = async (
```

Comment:
nit: prefer named function (`export async function setupContainer(`)

Reply:
Why is that?   I usually prefer arrow unless I explicitly want `this` binding.